seq2science/scripts/combine_biological_reps.py

```diff
@@ -0,0 +1,21 @@
+from io import StringIO
+from contextlib import redirect_stdout, redirect_stderr
+
+import pandas as pd
+
+with open(str(snakemake.log), "w") as f:
+   with redirect_stdout(f), redirect_stderr(f):
+      counts = pd.read_table(snakemake.input[0], comment="#", index_col=0)
+      samples = pd.read_table(StringIO(snakemake.params.samples), sep="\s+")
+      breps = snakemake.params.breps
+
```

Comment:
This will parse the samples file the same way (if the `descriptive_name` column is sparse, it will use replicate/sample names):
```python
from seq2science.utils import dense_samples

samples = dense_samples(samples, config)
```